guides/kfp-admin-guide.md

```diff
@@ -93,6 +106,8 @@ KFP Caching will cache all the workloads that use the same task with the same in
 kubectl set env -n kubeflow deploy/ml-pipeline CACHE_ENABLED=false
 ```
 
+<<<<<<< HEAD
+=======
 ## Change the Tekton Terminate API Method
```

Comment:
Resolve confict?

Reply:
Thanks. Sorry I forgot to save my changes in my previous commit. I fixed it now.